change/@fluentui-react-f8927acf-3db5-433e-bf67-dfc608a6ca4f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=23362&commitId=1454b386182c3280868b2520edee452150a3f6dd&env=prod&clientType=FLUENTUI)


  <details><summary><b>PeoplePicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.List disabled.chromium.png|492| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
The previous visual was wrong. The new visual is correct.